lectures/getting_started_julia/getting_started.md

```diff
@@ -41,138 +43,256 @@ While you will eventually use other editors, there are some advantages to starti
+```
 
-Then, you can run Jupyterlab using the Conda installation with
+(install_packages)=
+## Installing Packages
 
-```{code-block} none
-jupyter lab
+After you have the notebooks available, as in {ref}`above <clone_lectures>`, these lectures depend on functionality (like packages for plotting, benchmarking, and statistics) that are not installed with every Jupyter installation on the web.
+
+As discussed above, you can start this directly from the [command palette](command_palette) with `<Ctrl+Shift+P>` then typing part of the `> Julia: Start REPL` command.
+```{figure} /_static/figures/vscode_intro_2.png
+:width: 75%
 ```
 
-Or following [these instructions](https://julialang.github.io/IJulia.jl/dev/manual/running/) instructions if you didn't install Anaconda separately.
+1. Start a REPL  it may do an initial compilation of some background packages, but will then look something like
+ 
+   ```{figure} /_static/figures/vscode_intro_3.png
+   :width: 100%
+   ```
 
-Navigate to the location you stored the lecture notes, and open the {doc}`Interacting with Julia <../getting_started_julia/julia_environment>` notebook to explore this interface and start writing code.
+3. Next type `]` to enter the package mode, which should indicate that the local project is activated by changing the cursor to `(quantecon-notebooks-julia) pkg>`.
 
-#### Git Command Line Approach
+4. Type `instantiate` to install all of the packages required for these notes.
 
-If you do not wish to install the GitHub Desktop, you can get the notebooks using the Git command-line tool.
+   ```{figure} /_static/figures/vscode_intro_4.png
+   :width: 100%
+   ```
 
-Open a new terminal session and run
+This process will take several minutes to download and compile all of the files used by the lectures.
 
-```{code-block} none
-git clone https://github.com/quantecon/lecture-julia.notebooks
+```{attention}
+If the cursor is instead `(@v1.6) pkg>` then you may not have started the integrated terminal in the correct location, or you used an external REPL.  Assuming that you are in the correct location, if you type `activate .` in the package mode, the cursor should change to `(quantecon-notebooks-julia) pkg>` as it activates this project file.
+
+One benefit of using the integrated REPL is that it will set important options for launching Julia (e.g. the number of threads) and activate the local project files (i.e. the `Project.toml` file in the notebooks directory) automatically.  If you use an external REPL, you will need to set these manually.  Here you would want to run the REPL with `julia --project --threads auto`  to tell Julia to set the number of threads equal to your local machine's number of cores, and to activate the existing project.  See [here](starting-a-repl) for more details.
```

Comment:
@Alim-faraji  is this where you saw the issue?

Reply:
IT should be `repl_main` instead

Reply:
It is.

Reply:
Ok, I change it.